ui/src/pages/Logs.tsx

```diff
@@ -1,3 +1,134 @@
-export default function Logs() {
-  return <div>This is the logs page!</div>;
+import { TextField } from '@mui/material';
```

Comment:
We should be consistent in how we import MUI components.

Here you are destructuring from `@mui/material`, but below you are using path imports. This section of the docs explains the difference: https://mui.com/material-ui/guides/minimizing-bundle-size/

We should verify that our configuration correctly uses **tree-shaking** (so that unused components are not imported, increasing bundle size), and then use destructuring imports everywhere.